include/ml_dsa/internals/poly/sampling.hpp

```diff
@@ -103,27 +103,27 @@ expand_s(std::span<const uint8_t, 64> rho_prime, std::span<ml_dsa_field::zq_t, k
         const uint8_t t0 = buf_span[boff] & 0x0f;
         const uint8_t t1 = buf_span[boff] >> 4;
 
-        if constexpr (eta == 2u) {
+        if constexpr (eta_value == 2u) {
```

Comment:
No need to change this.

Reply:
I did change this to be 100% compliant with the original code, as you were using η and not eta

Reply:
`η` was uint32_t, `eta` was zq_t. It's not same now.